src/apps/account/models/user.ts

```diff
@@ -17,11 +23,33 @@ export class User extends BaseModel<User> {
   @prop({ required: false, limit: 100 })
   public email: string;
 
+  @prop({ required: true, limit: 100 })
+  public address: string;
+
+  @prop({ required: true, limit: 50 })
+  public city: string;
+
+  @prop({ required: true, limit: 50 })
+  public state: string;
+
+  @prop({ required: true, limit: 50 })
+  public zipCode: string;
+
+  @prop({ default: false })
+  public isMember: boolean;
+
   toEntity(user = new User()): User {
     user = this.toBaseEntity(user);
     user.firstName = this.firstName;
     user.lastName = this.lastName;
```

Comment:
firstName & lastName are not mentioned in the specs, therefore pls remove them out.